Gruntfile.js

```diff
@@ -93,7 +95,15 @@ module.exports = function (grunt) {
 					'!node_modules/nodebb-*/node_modules/**',
 					'!node_modules/nodebb-*/.git/**'
 				]
-			}
+			},
+			langUpdated: {
+				files: [
+					'public/language/**/*.json',
+					'node_modules/nodebb-*/**/*.json',
```

Comment:
I see that it's difficult to predict what the language folder would be here... can you add exclusions for `package.json` and `plugin.json`?

Reply:
Hi there @julianlam , I think `plugin.json` should be watched for changes, as developer tend to debug it.

Reply:
plugin.json is version controlled and should never be changed in our recommended setup. Any other setups that require this (e.g. Heroku) are outside of the norm and we shouldn't be changing behaviour to suit it.

Reply:
But I still have to try and try to config it right in the first place and refactor, add amd modules in the future. And I don't see watching for `plugin.json` will cost anything but deleting one line of code.